### ISSUE TITLE
Pairing KaKu device again after deleting them first time not working

### ISSUE THREAD

**Issue by pkonings** [athombv/homey#102]
After second setup homey yesterday and succesfully paring a KaKu remote (AGD2-3500R and a wallswitch device as well, I could not get these devices triggering the homey voice responding in a simple flow. (test button in flow screen worked, i.e. homey voice worked)

After deleting them as a attempt to start again I cannot pair them anymore.....

**Comment by pkonings** [comment]
Update: I just reconnected power - where is the shutdown/restart functionality? - and paring seems to work again.

**Comment by bvdbos** [comment]
same issue as #101 ?

**Comment by pkonings** [comment]
Update 2: flow When:[kaku remote "a button is pressed"] => Then: ["say something"] is not working

![image](https://cloud.githubusercontent.com/assets/10164680/12976172/7400d040-d0c2-11e5-91f5-b656c5d256be.png)

**Comment by pkonings** [comment]
@bvdbos : not same issue as #101. Procedure to add KaKu remote and wallswitch to devices was stuck because Homey did not react on pressing button on remote device. As said, after hard reboot it did.

**Comment by elgenioNL** [comment]
Had this problem also yesterday, no kaku devices of impuls devices working in flow, or possible to learn a new device. 
Had to do a powercycle to fix it. Seems that the 433 part stops working sometimes.

**Comment by hschenkelaars** [comment]
Same here. When rebooting homey my 433 isn't working anymore. As bvdbos stated a powercycle solves the issue. So if you need to reboot you should powercycle homey for 433 to keep working.

**Comment by pkonings** [comment]
I had to move the homey this evening to another place within the living room causing it to be reconnected to another powersocket. Just before that I succesfully added two lightswitches. After reconnecting they didn't work anymore, so I decided to unplug the power and plug it in again. 
Magically the lightswitches worked again, i.e. using the screeninterface cards.
I als changed my KakU remote within the previous 'flow'-screenshot above to the following and that change works....
![image](https://cloud.githubusercontent.com/assets/10164680/12990715/11f35b12-d10c-11e5-94bf-285f0605a648.png)
